bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/SitemapResource.java

```diff
@@ -247,9 +247,13 @@ public Response getPageData(@Context HttpHeaders headers,
     @POST
     @Path(SEGMENT_EVENTS + "/subscribe")
     @ApiOperation(value = "Creates a sitemap event subscription.")
-    @ApiResponses(value = { @ApiResponse(code = 201, message = "Subscription created.") })
+    @ApiResponses(value = { @ApiResponse(code = 201, message = "Subscription created."),
+            @ApiResponse(code = 403, message = "Subscriptions limit reached.") })
     public Object createEventSubscription() {
         String subscriptionId = subscriptions.createSubscription(this);
+        if (subscriptionId == null) {
+            return JSONResponse.createResponse(Status.FORBIDDEN, null, "Max number of subscriptions is reached.");
```

Comment:
I would still prefer `429` (WRT the comment I read and linked).
But to use a custom code needs another little change, so we can IMHO go for that ATM.

Reply:
I'd agree that FORBIDDEN is not the correct return status. I wouldn't consider it a client error (4xx) at all, rather a server error (5xx). I would vote for "503 SERVICE UNAVAILABLE: The server is currently unable to handle the request due to a temporary overload".